inspire/dojson/hep/schemas/hep-0.0.1.json

```diff
@@ -593,6 +612,45 @@
                 }
             }
         },
+        "succeeding_entry": {
+            "type": "object",
+            "properties": {
+                "relationship_code": {
+                    "type": "string"
+                },
+                "recid": {
+                    "type": "string"
+                },
+                "isbn": {
+                    "type": "string"
+                }
+            }
+        },
+        "acquisition_source": {
+            "type": "object",
+            "properties": {
+                "source": {
+                    "type": "string",
+                    "description": "Specific volume number"
+                },
+                "email": {
+                    "type": "string",
+                    "description": "Title of the book series"
```

Comment:
Yeah, this is a copy-paste slip. The actual book_series field is described here: 
https://github.com/jalavik/inspire-next/blob/bibfield-dojson/inspire/dojson/hep/schemas/hep-0.0.1.json#L602-L614